docs/pages/access-controls/sso.mdx

```diff
@@ -328,7 +328,11 @@ spec:
   # in the Web interface
   display: "Okta"
 
-  acs: https://teleport-proxy.example.com:3080/v1/webapi/saml/acs
+  # enables/disables idp-initiated saml login
+  allow_idp_initiated: false
+
+  # The last segment of the URL must be identical to the connector metadata name.
```

Comment:
Is this a new requirement, or it just wasn't documented before?

Reply:
This is a new requirement, existing connectors don't break, but IdP initiated logins will not work without it and will return an error code noting that the connector name isn't part of the ACS callback URI.

Reply:
So does this new requirement only apply when enabling IdP-initiated login, or all the time going forward?

Reply:
It's only required with IdP-initiated login, but I'd say it's good practice to act like you should do it for simplicity's sake.

Reply:
Okay, if we're introducing new constraints I think we need to mention it more than just a comment in the config YAML. In fact, a community user recently reported this error in slack and wasn't sure what do to about it.

Reply:
I have updated relevant guides to fulfill this constraint so you'll get this done correctly when you set up a new connector. Should we write an additional note/section about enabling this for existing connectors too?

Reply:
As discussed in our 1x1, I’ll be adding validation on connector upsert to ensure this cannot be violated. Docs should be good as-is.